es/lessons/basics/collections.md

```diff
@@ -146,21 +159,13 @@ iex> %{foo: "bar", hello: "world"} == %{:foo => "bar", :hello => "world"}
 true
 ```
 
-## Diccionarios
-
-En Elixir, ambos, listas de palabras claves y mapas implementan el módulo `Dict`; tales se conocen colectivamente como diccionarios. Si necesitas construir tu propio almacenamiento clave-valor, implementar el módulo `Dict` es un buen lugar para empezar.
-
-El [módulo `Dict`](http://elixir-lang.org/docs/stable/elixir/#!Dict.html) provee un número de funciones útiles para interactuar y manipular esos diccionarios:
+Otra característica interesante de los mapas es que poseen su propia sintaxis para actualizar y acceder los átomos como clave.
```

Comment:
It is  "__acceder a__ los átomos", but still this is not clear. 
How about: 
> Otra característica interesante de los mapas es que poseen su propia sintaxis para realizar operaciones de actualización y lectura.

I wouldn't mention "atom keys". It is a bit confusing that little bit at the end.